src/pages/tests/UserSettings.spec.js

```diff
@@ -0,0 +1,88 @@
+// Configurations
+import { mount, config } from "@vue/test-utils";
+import { createTestingPinia } from "@pinia/testing";
+
+import NavBar from "@/components/NavBar.vue";
+import NavFooter from "@/components/NavFooter.vue";
+import router from "@/router/router.js";
+
+config.global.components["nav-bar"] = NavBar;
+config.global.components["nav-footer"] = NavFooter;
+config.global.plugins.push(router);
+
+// Test
+import {
+    describe,
+    it,
+    expect,
+    beforeEach
+} from "vitest";
+
+import UserSettings from "@/pages/UserSettings.vue";
+import { useUserStore } from "@/store/user/userStore.js";
+
+
+describe("UserSettings", () => {
+    let inputs;
+    let userObj;
+    let wrapper;
+
+    beforeEach(() => {
```

Comment:
Idk about this why render it every time if we only are checking for static info from very different parts of the DOM